requirements.txt

```diff
@@ -6,3 +6,4 @@ uwsgi==2.0.10
 psycopg2==2.6
 tox==1.9.2
 static3==0.5.1
+-e git+https://github.com/mitodl/xbundle.git@7799a0957c502103fb030fc0b16c64204f551c75#egg=xbundle
```

Comment:
Sorry for the trifle, you shouldn't need to use -e here, i.e. https://github.com/mitodl/lore/blob/master/test_requirements.txt#L6